android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
```

Comment:
Android requires this configuration to tell which domains / IP addresses are allowed to be accessed via HTTP.
In my dev. environment I use HTTP instead of HTTPS because it's easier than setting up an HTTPS proxy to access the API. So, I just put my IP address in this file and android will allow the app to communicate with the api using HTTP.

```
<network-security-config>
    <domain-config cleartextTrafficPermitted="true">
        <domain includeSubdomains="true">192.168.1.73</domain>
    </domain-config>
</network-security-config>
```

Reply:
because HTTP will not be allow, I'll revert these changes.